doc/src/manual/functions.md

```diff
@@ -53,12 +53,13 @@ julia> ∑(2, 3)
 
 ## Argument Passing Behavior
 
-Julia function arguments follow a convention sometimes called "pass-by-sharing", which means that
-values are not copied when they are passed to functions. Function arguments themselves act as
-new variable *bindings* (new locations that can refer to values), but the values they refer to
-are identical to the passed values. Modifications to mutable values (such as `Array`s) made within
-a function will be visible to the caller. This is the same behavior found in Scheme, most Lisps,
-Python, Ruby and Perl, among other dynamic languages.
+Julia function arguments follow a convention sometimes called "pass-by-sharing" or
+"pass-by-reference". This means that values are not copied when they are passed to
```

Comment:
This is not a correct changes since "pass by reference" traditionally means something very different, although people do, confusingly use the term to mean "pass by sharing".